rwatch/ui/action_menu.c

```diff
@@ -73,7 +74,7 @@ ActionMenuItem * action_menu_level_add_action(ActionMenuLevel * level, const cha
     new_item->cb = cb;
     new_item->action_data = action_data;
     
-    if (*level->capacity == level->count)
+    if (*level->capacity == level->count) //XXX
```

Comment:
`capacity` should not be a pointer, much more of a reason to *not* dereference it